sumy/nlp/stemmers/greek.py

```diff
@@ -0,0 +1,43 @@
+"""The code of the stem_word function relies on the greek stemmer found in this python module
```

Comment:
I am quite nervous from this. I am OK with including optional GPL library in the code and it's fine but here you suggest you copy-pasted this code from the library. Is that true? Can you send me a link to the file you are mentioning here? Because if this file falls under GPL I think the whole sumy project would have to be GPL and that is not what I want.

https://opensource.stackexchange.com/questions/12064/which-license-for-a-project-combining-mit-libraries-and-gpl-modified-files